frontend-react/src/components/Content/MarkdownLayout.tsx

```diff
@@ -55,7 +55,7 @@ export function MarkdownLayout({
             }}
             {...mdx}
         >
-            <GridContainer className="usa-prose">
+            <GridContainer className="usa-prose tablet:margin-top-6 margin-bottom-5">
```

Comment:
I feel this may be approaching the problem from the wrong side. The root page heading should have bottom margin spacing so that all content afterwards site-wide is spaced as desired. This may require adjusting edge cases (home page content header may need negative top margin adjustment to push back towards root header).

Reply:
@jpandersen87  In  that case, should this style be removed from the IASideNavTemplate as well?

Reply:
Yep.